editor/src/clj/editor/app_view.clj

```diff
@@ -2619,3 +2623,52 @@ If you do not specifically require different script states, consider changing th
   (enabled? [] (disk-availability/available?))
   (run [app-view changes-view prefs workspace project]
        (ensure-exists-and-open-for-editing! shared-editor-settings/project-shared-editor-settings-proj-path app-view changes-view prefs project)))
+
+(defn get-linux-desktop-entry [launcher-path install-dir]
```

Comment:
This function could also be made private, I guess there won't be used anywhere else